tests/e2e-evm/test/abi_disabled.test.ts

```diff
@@ -165,14 +195,16 @@ describe("ABI_DisabledTests", function () {
             let revertDetail = "";
             try {
               await call;
-            } catch(e) {
+            } catch (e) {
+              expect(e).to.be.instanceOf(BaseError, "expected error to be a BaseError");
+
               if (e instanceof BaseError) {
                 revertDetail = e.details;
               }
             }
 
-            const expectedMatch = /call not allowed to disabled contract/;
-            expect(expectedMatch.test(revertDetail), `expected ${revertDetail} to match ${expectedMatch}`).to.be.true;
+            const expectedMatch = "call not allowed to disabled contract";
+            expect(revertDetail).to.equal(revertDetail, `expected ${revertDetail} to match ${expectedMatch}`);
```

Comment:
Nice simplification -- we *may* have to change it back at some point though, but let's keep for now.

The error response of viem for hardhat, kvtool, etc can be a bit different so I meant to only match the revert message here.  Unfortunately, chai doesn't have a great matcher for regex like testify.